src/plugins/spark/features/Balancer/getters.ts

```diff
@@ -1,8 +1,21 @@
-import { blockById } from '@/plugins/spark/store/getters';
+import { blockValues, blockById } from '@/plugins/spark/store/getters';
 import { RootStore } from '@/store/state';
 import { BalancerBlock } from './state';
 
 export const typeName = 'Balancer';
 
 export const getById = (store: RootStore, serviceId: string, id: string) =>
   blockById<BalancerBlock>(store, serviceId, id, typeName);
+
+export const getClients = (store: RootStore, serviceId: string, balancerId: string) => {
```

Comment:
This can be implemented a lot cleaner using lodash get https://lodash.com/docs/4.17.11#get